src/Kinepolis/KinepolisService.php

```diff
@@ -58,6 +63,34 @@ public function import(): void
             $this->logger->error('Problem with Kinepolis Service Authentication: ' . $exception->getMessage());
             return;
         }
+
+        try {
+            $theaters = $this->client->getTheaters($token);
+        } catch (\Exception $exception) {
+            $this->logger->error('Problem with fetching theatres from Kinepolis Service: ' . $exception->getMessage());
+            return;
+        }
+
+        /**
+         * @var ParsedPriceForATheater[] $parsedPrices
+         */
+        $parsedPrices = [];
+        foreach ($theaters as $theater) {
+            try {
+                $pricesToParse = $this->client->getPricesForATheater($token, $theater['tid']);
+            } catch (\Exception $exception) {
+                $this->logger->error('Problem with fetching the prices from Theater ' . $theater['tid'] . ': ' . $exception->getMessage());
```

Comment:
Really not a fan of the  $theater['tid'] structure, would move this to a DTO

Reply:
No strong opinion on this one. Because it will always be a conversion from an array to something more typed and that is now done inside the parsers. This way we have a nice separation for concerns. The API client just gets the arrays, the parsers convert them and the service binds it all together.

Reply:
There is a `ParsedMovie` class.

Reply:
And a `ParsedPriceForATheater` which is the only thing we need from the KinepolisAPI.
All necessary information for a Theater are stored inside the Places, which are already in UiTdatabank.
Only the Price is Needed because there format with surcharges depending on the length or dimension is incompatible with our PriceInfo format.